doc/source/10min.rst

```diff
@@ -31,9 +14,22 @@ Customarily, we import as follows:
 
 .. ipython:: python
 
-   import pandas as pd
    import numpy as np
-   import matplotlib.pyplot as plt
+   import pandas as pd
+
+.. ipython:: python
+   :suppress:
```

Comment:
why not at the beginning and with numpy and pandas imports in the same suppress block? I think this is what we do in all the other pages, right?